reference/best_practices/reusable_bundles.rst

```diff
@@ -1,4 +1,95 @@
 Reusable Bundles for Akeneo PIM community
 =========================================
 
-Coming soon!!! :)
+You're integrator and you want to share your project with the awesome Akeneo PIM community?
+Let's read our advices to make a reusable and documented bundle!
+
+
+What do you want when you are creating a reusable bundle?
+
+* **Compatibility**: fit Akeneo PIM features
+* **Maintainability**: make your code flexible and testable (and tested)
+* **Extensibility**: to allow integrators to fit your bundle with their specific needs
+* **Scalability**: know your application limits and document them
+
+
+Compatibility
+-------------
+
+Your bundle won't be reusable for some customers if you forgot some cases. But that's not a problem if they are aware of the compatibility of your bundle.
+
+Akeneo PIM storage stack comes with MySQL and hybrid MySQL/MongoDB (:doc:`/reference/technical_information/choose_database`).
+Use the ProductQueryBuilder which is an overlayer making work both storages with the same API.
+The following cookbook has been done to help you: ":doc:`/cookbook/catalog/product/query`".
+
+Two existing distributions CE/EE. The main project is `Akeneo PIM Community Edition`_ (CE) but
+we also have an `Enterprise Edition`_ (EE) which comes with a bunch of extra features dedicated to advanced needs.
+
+Many attribute types are managed (14 in CE and one more in EE). Be sure you don't forget a use case for an attribute type.
+
+Try your bundle on another catalogs. There is a multiplicity of catalog possibilities.
```

Comment:
Use:  "*with another catalog*" or "*with other catalogs*". Next sentence is not helpful, of course there are hundreds of different catalogs ;)